app/src/main/java/io/github/droidkaigi/confsched/viewmodel/event/EventBus.java

```diff
@@ -0,0 +1,24 @@
+package io.github.droidkaigi.confsched.viewmodel.event;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+import rx.subjects.SerializedSubject;
+import rx.subjects.Subject;
+
+public class EventBus {
```

Comment:
Usually yes.
Because ViewModel should not depends View, then ViewModel send "signal" and View receive it.
.NETer are calling it "Messenger".
Any MVVM frameworks have Messenger service.

Ex.

* [いまさら聞けない「MVVM + Messenger パターン」超入門 - present](http://tnakamura.hatenablog.com/entry/20110218/mvvm_messenger)
* [MessengerでViewModelからViewに通知する | garicchi.com](http://garicchi.com/?p=7971)

Frequently used cases are

* View navigation
* show Dialog, Toast

You should be careful using EventBus, consider using ViewModel's ObservableField first.